crates/evm/execution-errors/src/lib.rs

```diff
@@ -75,12 +75,19 @@ pub enum BlockValidationError {
         /// The error message.
         message: String,
     },
-    /// EVM error during withdrawal requests contract call
+    /// EVM error during withdrawal requests contract call [EIP-7002]
+    ///
+    /// [EIP-7002]: https://eips.ethereum.org/EIPS/eip-7002
     #[error("failed to apply withdrawal requests contract call: {message}")]
     WithdrawalRequestsContractCall {
         /// The error message.
         message: String,
     },
+    /// Error when decoding deposit requests from receipts [EIP-6110]
+    ///
+    /// [EIP-6110]: https://eips.ethereum.org/EIPS/eip-6110
+    #[error(transparent)]
+    DepositRequestDecode(#[from] alloy_sol_types::Error),
```

Comment:
I'd like to avoid this because I want to keep the errors as lean as possible, although this is already a bit bloated...

imo we can use the string representation of alloy_sol_types::Error here